src/components/QuicksightDashboard.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import {
+  QuickSightEmbeddedURLData,
+  LibrariesData,
+  LibraryData,
+} from "../interfaces";
+import { Store } from "redux";
+import { RootState } from "../store";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+
+export interface QuicksightDashboardStateProps {
+  isFetchingLibraries?: boolean;
+  libraries?: LibraryData[];
+}
+
+export interface QuicksightDashboardDispatchProps {
+  fetchLibraries?: () => Promise<LibrariesData>;
+  fetchQuicksightEmbedUrl?: (
+    dashboardId: string,
+    ld: LibrariesData
+  ) => Promise<QuickSightEmbeddedURLData>;
+}
+
+export interface QuicksightDashboardOwnProps {
+  store?: Store<RootState>;
+  dashboardId?: string;
```

Comment:
We won't need `dashboardId` as a prop here, assuming that we stop passing the hardcoded value from the parent page.